cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-what-is-rosa.adoc

```diff
@@ -14,18 +14,174 @@ Red Hat OpenShift Service on AWS (ROSA) is a fully-managed turnkey application p
+
 A list of the account-wide and per-cluster roles is provided in the xref:../../rosa_architecture/rosa-sts-about-iam-resources.adoc#rosa-sts-account-wide-roles-and-policies-creation-methods_rosa-sts-about-iam-resources[ROSA documentation].
 
+== Administrators
+A ROSA customer's administrator can manage users and quotas in addition to accessing all user-created projects.
+
+== OpenShift versions and upgrades
+ROSA is a managed service which is based on OpenShift Container Platform. You can view the current version and life cycle dates in the xref:../../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle-dates_rosa-life-cycle[ROSA documentation].
+
+Customers can upgrade to the newest version of OpenShift and use the features from that version of OpenShift. For more information, see xref:../../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle-dates_rosa-life-cycle[life cycle dates]. Not all OpenShift features are be available on ROSA. Review the xref:../../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc#rosa-service-definition[Service Definition] for more information.
+
+== Support
+ROSA is supported by Red Hat and AWS, and you can contact the support team from either company to begin troubleshooting. Any escalations will be facilitated by Red Hat and AWS to engage the best team to address the issue.
+
+* link:https://access.redhat.com/[Red Hat Support]
+* link:https://aws.amazon.com/premiumsupport/[AWS Support]
+** AWS support customers must have a valid AWS support contract
+
+You can also visit the link:http://access.redhat.com/[Red Hat Customer Portal] to search or browse through the Red Hat knowledge base of articles and solutions relating to Red Hat products or submit a support case to Red Hat Support.
+
+You can also open a ticket directly from the link:https://console.redhat.com/openshift[Red Hat OpenShift Cluster Manager]. See the xref:../../support/getting-support.adoc#getting-support[ROSA support documentation] for more details about obtaining support.
+
+If a ROSA cluster is not upgraded before the "end of life" date, the cluster continues to operate in a limited support status. The SLA for that cluster will no longer be applicable, but you can still get support for that cluster. See the xref:../../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle[limited support status]documentation for more details.
+
+== Service-level agreement (SLA)
+Refer to the xref:../../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc#rosa-sdpolicy-sla_rosa-service-definition[ROSA SLA] page for details.
+
+== Notifications and communication
+Red Hat will provide notifications regarding new features, updates, and scheduled maintenance through email, including AWS updates and email,  and the Red Hat console service log.
```

Comment:
```suggestion
Red Hat will provide notifications regarding new Red Hat and AWS features, updates, and scheduled maintenance through email  and the Red Hat console service log.
```

Reply:
Not sure if this is now accurate (although I think it is) but current phrasing is confusing IMO.